app/assets/stylesheets/shared/bootstrap-variables.scss

```diff
@@ -73,3 +73,5 @@ $form-floating-input-padding-t: $input-btn-padding-y;
 $form-floating-input-padding-b: $input-btn-padding-y;
 $form-floating-label-opacity: 1;
 $form-floating-label-transform: scale(0.85) translateY(-1.75rem) translateX(0.15rem);
+
+$border-radius: 3px;
```

Comment:
Just noting that the base bootstrap variables define these [in rems](https://github.com/twbs/bootstrap-rubygem/blob/main/assets/stylesheets/bootstrap/_variables.scss#L495).  But if you're cool with pixels, so am I!

Reply:
fixed